psydac/api/tests/test_assembly.py

```diff
@@ -443,6 +444,32 @@ def test_non_symmetric_BilinearForm(backend):
 
     print("PASSED")
 
+#==============================================================================
+def test_non_symmetric_different_space_BilinearForm(backend):
+
+    kwargs = {'backend': PSYDAC_BACKENDS[backend]} if backend else {}
+
+    domain = Square()
+    V = VectorFunctionSpace('V', domain, kind='Hdiv')
+    X   = VectorFunctionSpace('X', domain, kind='h1')
+
+    u    = element_of(X, name='u')    
+    w   = element_of(V, name='w')
+
+    A   = BilinearForm((u,w), integral(domain, Inner(u,w)))
+
+    ncells = [4, 4]
+    degree = [2, 2]
+
+    domain_h = discretize(domain, ncells=ncells)
+    Vh = discretize(V, domain_h, degree=degree)
+    Xh  = discretize(X, domain_h, degree=degree)
+
+    ah  = discretize(A, domain_h, (Xh,Vh), **kwargs)
```

Comment:
Improve white spaces:
```suggestion
    Xh = discretize(X, domain_h, degree=degree)

    ah = discretize(A, domain_h, (Xh, Vh), **kwargs)
```